fe/fe-core/src/main/java/com/starrocks/persist/EditLog.java

```diff
@@ -1828,4 +1851,20 @@ public void logAlterTableProperties(ModifyTablePropertyOperationLog info) {
     private void logJsonObject(short op, Object obj) {
         logEdit(op, out -> Text.writeString(out, GsonUtils.GSON.toJson(obj)));
     }
+
+    public void logSetDefaultStorageVolume(SetDefaultStorageVolumeLog info) {
+        logEdit(OperationType.OP_SET_DEFAULT_STORAGE_VOLUME, info);
+    }
+
+    public void logCreateStorageVolume(StorageVolume storageVolume) {
+        logEdit(OperationType.OP_CREATE_STORAGE_VOLUME, storageVolume);
+    }
+
+    public void logUpdateStorageVolume(StorageVolume storageVolume) {
+        logEdit(OperationType.OP_UPDATE_STORAGE_VOLUME, storageVolume);
+    }
+
+    public void logDropStorageVolume(DropStorageVolumeLog info) {
```

Comment:
DropStorageVolumeLog and SetDefaultStorageVolumeLog are just a string field persistency, can we use a generic StringWritable object to serialize/deserialize the field, instead of creating each single Class for the serialize/deserialize?